### ISSUE TITLE
Issue parsing floats in GFF fields

### ISSUE THREAD

**Issue by th3w1zard1** [NickHugi/PyKotor#5]
Using PyKotor to attempt to install the Juhani Romance Enhancement mod, I get this error:
```
[7/2/2023 6:59:31 PM] Using changes.ini path: C:\Users\*****\Documents\k1 mods\Juhani_Romance_Enhancement\tslpatchdata\changes.ini

[7/2/2023 6:59:32 PM] Traceback (most recent call last):
  File "pykotorcli.py", line 66, in <module>
  File "pykotor\tslpatcher\config.py", line 118, in install
  File "pykotor\tslpatcher\config.py", line 113, in config
  File "pykotor\tslpatcher\config.py", line 77, in load
  File "pykotor\tslpatcher\reader.py", line 56, in load
  File "pykotor\tslpatcher\reader.py", line 189, in load_gff
  File "pykotor\tslpatcher\reader.py", line 331, in add_field_gff
  File "pykotor\tslpatcher\reader.py", line 294, in add_field_gff
ValueError: could not convert string to float: '113,430000305176'
```

Relevant `changes.ini` snippet:
```
[m44ac.git]
AddField0=gff_m44ac_TriggerList_18_0
AddField1=gff_m44ac_CameraList_22_0
AddField2=gff_m44ac_CameraList_23_0
!Destination=Modules\unk_m44ac.mod
[gff_m44ac_TriggerList_18_0]
FieldType=Struct
Path=TriggerList
Label=
TypeId=1
AddField0=gff_m44ac_TemplateResRef_0
AddField1=gff_m44ac_XPosition_0
AddField2=gff_m44ac_YPosition_0
AddField3=gff_m44ac_ZPosition_0
AddField4=gff_m44ac_XOrientation_0
AddField5=gff_m44ac_YOrientation_0
AddField6=gff_m44ac_ZOrientation_0
AddField7=gff_m44ac_Geometry_0
[gff_m44ac_TemplateResRef_0]
FieldType=ResRef
Label=TemplateResRef
Value=jre_trigger
[gff_m44ac_XPosition_0]
FieldType=Float
Label=XPosition
Value=113,430000305176
[gff_m44ac_YPosition_0]
FieldType=Float
Label=YPosition
Value=69,6999969482422
[gff_m44ac_ZPosition_0]
FieldType=Float
Label=ZPosition
Value=13,7505826950073
```

Perhaps the comma simply needs to be treated as a dot and handle both cases?

**Comment by th3w1zard1** [comment]
[changes.zip](https://github.com/NickHugi/PyKotor/files/11931799/changes.zip)

**Comment by th3w1zard1** [comment]
Fixed by #7
